blivet/devicefactory.py

```diff
@@ -277,6 +277,7 @@ class DeviceFactory(object):
                          "container_size": SIZE_POLICY_AUTO,
                          "container_raid_level": None,
                          "container_encrypted": None}
+    _device_min_size = Size(0)  # no limit by default, limited only by filesystem size
```

Comment:
I added this for tests. I'm not sure if we want to make this "public" for all devices/factories, Anaconda might need a new API for getting device size limits (only limits we have now are based on filesystem min size).